templates/blog/index.html

```diff
@@ -5,42 +5,36 @@
   <link rel="alternate" type="application/rss+xml"  href="/blog/feed" title="Canonical.com's Blog RSS feed">
 {% endblock %}
 
-{% block content%}
+{% set is_paper = True %}
 
-  {% if current_page and current_page == 1 and featured_articles %}
-    {% include "blog/featured-articles.html" %}
-  {% endif %}
-
-  <section class="p-strip is-shallow" id="posts-list">
-    {% with topic="" %}{% include "blog/filters.html" %}{% endwith %}
-    <div class="row u-equal-height u-clearfix">
-    {% for article in articles %}
-      {% if loop.index in [1,2] %}
-        {% with summary_visible=True %}{% include "blog/blog-card.html" %}{% endwith %}
-      {% elif loop.index == 3 %}
-        <div class="col-4">
-          {% include 'blog/newsletter-form.html' %}
+{% block content %}
+<div class="p-strip">
+  <div class="row">
+    <div class="col-9 col-start-large-4 col-medium-4 col-start-medium-3">
+      <div class="row p-section">
```

Comment:
Nested rows here seems wrong

Reply:
don't think we can avoid them, it is how the grid works when nesting is required - this is documented behaviour: https://vanillaframework.io/docs/patterns/grid#nested-columns